Drivers/Src/MotorController.c

```diff
@@ -6,7 +6,7 @@
 #define SUCCESS     1
 #define FAILURE     0
```

Comment:
These can be removed now also. Sorry didn't catch this earlier

Reply:
Fixed